src/Imdb/Title.php

```diff
@@ -1126,6 +1126,52 @@ public function photo_localurl($thumb = true)
         }
         return false;
     }
+    
+    #-------------------------------------------------[ Main images on title page ]---
+
+    /** Get URLs for the 12 pictures on the title page
+     * @return array [0..n] of [imgsrc, imglink, bigsrc], where
+     *    imgsrc is the URL of the thumbnail IMG as displayed on main page.
+     *    imglink is the link to the page with the "big image".
+     *    bigsrc is the URL of the "big size" image itself.
+     * @author moonface
+     * @author izzy
+     * @author Ed
+     */
+    public function mainPictures()
+    {
+        if (empty($this->main_pictures)) {
+            $xpath = $this->getXpathPage("Title");
+            $cells = $xpath->query("//div[@class='ipc-photo ipc-photo--base ipc-photo--dynamic-width photos-image ipc-sub-grid-item ipc-sub-grid-item--span-2']");
```

Comment:
Are the a more absolute class/place to look? If they just change the order of one of these, it will break (we have no tests).

Reply:
This is the only classname that is unique enough and works, i can't find any better one, do you?

Reply:
Something like this? It will first look up "Photos" header. Go one step up and therefor look for all images.

```php
if ($header = $xpath->query('//div[@data-testid="photos-header"]')) {
    if ($header->length > 0) {
        $cells = $xpath->query('.//div[contains(@class, "photos-image")]', $header[0]->parentNode);
        // rest of code
    }
}
```

Currently "photos-image" are only applied to these images, so we don't need to check for header, but that's there for safety.